codewof/static/css/third-party/introjs.css

```diff
@@ -0,0 +1,506 @@
+.introjs-overlay {
```

Comment:
Does this sentence in our README apply regarding this file (and/or the other one)?

> If a source file of a third-party library or system is stored within this repository, then a copyright notice should be present at the top of the file.